lib/code_climate/test_reporter/git.rb

```diff
@@ -44,7 +44,8 @@ def head_from_git
         end
 
         def head_from_ci
-          Ci.service_data[:commit_sha]
+          sha = Ci.service_data[:commit_sha]
+          return nil if sha.empty?
```

Comment:
Right you are. So weird.. though I ran this from `pry` and it returned the sha. I'll be explicit about it. 👍